README.md

```diff
@@ -78,9 +78,33 @@ SYNOPSIS
 
 POSITIONAL ARGUMENTS
     TEXT
-        (required) A phrase less than 77 characters which you would like to visualize.
+        (required for CLI) A phrase less than 77 characters which you would like to visualize.
 
 FLAGS
+    --img=IMAGE_PATH
```

Comment:
Can you add these new parameter descriptions to the docstring in cli.py as well? lucid is using a library called fire to generate the CLI tool and it uses the docstrings to generate its `--help`

Reply:
Hm, it actually appears that cli.py also just needs these params to be passed in in the first place. As it stands, they're just gonna be the defaults.

Reply:
Sure, I'll update the cli.py when I have the time to do so, maybe during the weekend.

Reply:
I have time to work on this now if you haven't already done so.

Reply:
https://github.com/NotNANtoN/deep-daze/pull/1/files
Implemented all the new parameters on the cli.